Add the buildout option mr.developer-verbose

sunew opened the pull request:
that enables showing the same out when running buildout as when running ./bin/develop up -v.

Respect the buildout -v setting for updates, just as it already does for checkouts.

Valuable in many cases to see the output of the git command - to see is actually updated.

sunew commented:
Thanks - and sure:)
Test running on master: 9 
On my feature branch: 9.
So do not break any tests.
I must admit its hard for me to see how to write a test for the new option. Input appreciated.

And: this is currently only for git.

mauritsvanrees commented:
I don't really see any extra output, but maybe I am testing it in the wrong way. Code looks good. I will merge.

mauritsvanrees commented:
I have released 1.36 with this. Thanks!

sunew commented:
Thanks! Just updated a local buildout on my side with the new release.

a) you need some activated development eggs controlled by mr.developer.
b) You should see some output like the following, when running buildout -v (or when using the property mr.developer-verbose = true):

    INFO: Updated 'plone.restapi' with git.
    * master
      remotes/origin/HEAD -> origin/master
      remotes/origin/barcelona-mocks
      remotes/origin/docs-formatting-spelling
      remotes/origin/docs-request-examples
      remotes/origin/expansion
      remotes/origin/failing-test-because-of-undocumented-changes
      remotes/origin/fix_frombaseunit
      remotes/origin/global_allow
      remotes/origin/issue-104-tisto
      remotes/origin/issue-158-password-reset
      remotes/origin/issue-159
      remotes/origin/issue-198-types-visibility
      remotes/origin/issue-212-vocabularies-endpoint
      remotes/origin/issue-38-sharing
      remotes/origin/jsonschema
      remotes/origin/master
      remotes/origin/object_field_serializer
      remotes/origin/ramon
      remotes/origin/robot
      remotes/origin/searchusers
      remotes/origin/sharing
      remotes/origin/sneridagh-groups
      remotes/origin/upgrade-flake8
      remotes/origin/uxf
    Your branch is behind 'origin/master' by 8 commits, and can be fast-forwarded.
      (use "git pull" to update your local branch)
    * master
      remotes/origin/HEAD -> origin/master
      remotes/origin/barcelona-mocks
      remotes/origin/docs-formatting-spelling
      remotes/origin/docs-request-examples
      remotes/origin/expansion
      remotes/origin/failing-test-because-of-undocumented-changes
      remotes/origin/fix_frombaseunit
      remotes/origin/global_allow
      remotes/origin/issue-104-tisto
      remotes/origin/issue-158-password-reset
      remotes/origin/issue-159
      remotes/origin/issue-198-types-visibility
      remotes/origin/issue-212-vocabularies-endpoint
      remotes/origin/issue-38-sharing
      remotes/origin/jsonschema
      remotes/origin/master
      remotes/origin/object_field_serializer
      remotes/origin/ramon
      remotes/origin/robot
      remotes/origin/searchusers
      remotes/origin/sharing
      remotes/origin/sneridagh-groups
      remotes/origin/upgrade-flake8
      remotes/origin/uxf
    Updating afb0142..8818b21
    Fast-forward
     CHANGES.rst                                        |   6 +
     docs/source/_json/groups.json                      |  48 ++++++
     docs/source/_json/groups_created.json              |  15 ++
     docs/source/_json/groups_delete.json               |   5 +
     .../source/_json/groups_filtered_by_groupname.json |  16 ++
     docs/source/_json/groups_get.json                  |  14 ++
     docs/source/_json/groups_update.json               |   5 +
     docs/source/groups.rst                             | 189 +++++++++++++++++++++
     src/plone/restapi/serializer/configure.zcml        |   1 +
     src/plone/restapi/serializer/group.py              |  31 ++++
     src/plone/restapi/services/configure.zcml          |   1 +
     src/plone/restapi/services/groups/__init__.py      |   0
     src/plone/restapi/services/groups/add.py           |  72 ++++++++
     src/plone/restapi/services/groups/configure.zcml   |  38 +++++
     src/plone/restapi/services/groups/delete.py        |  50 ++++++
     src/plone/restapi/services/groups/get.py           |  87 ++++++++++
     src/plone/restapi/services/groups/update.py        |  70 ++++++++
     src/plone/restapi/tests/test_documentation.py      |  97 +++++++++++
     src/plone/restapi/tests/test_serializer_group.py   |  42 +++++
     src/plone/restapi/tests/test_services_groups.py    | 165 ++++++++++++++++++
     src/plone/restapi/tests/test_types.py              |  69 +++++++-
     src/plone/restapi/types/utils.py                   |  12 ++
     22 files changed, 1032 insertions(+), 1 deletion(-)
     create mode 100644 docs/source/_json/groups.json
     create mode 100644 docs/source/_json/groups_created.json
     create mode 100644 docs/source/_json/groups_delete.json
     create mode 100644 docs/source/_json/groups_filtered_by_groupname.json
     create mode 100644 docs/source/_json/groups_get.json
     create mode 100644 docs/source/_json/groups_update.json
     create mode 100644 docs/source/groups.rst
     create mode 100644 src/plone/restapi/serializer/group.py
     create mode 100644 src/plone/restapi/services/groups/__init__.py
     create mode 100644 src/plone/restapi/services/groups/add.py
     create mode 100644 src/plone/restapi/services/groups/configure.zcml
     create mode 100644 src/plone/restapi/services/groups/delete.py
     create mode 100644 src/plone/restapi/services/groups/get.py
     create mode 100644 src/plone/restapi/services/groups/update.py
     create mode 100644 src/plone/restapi/tests/test_serializer_group.py
     create mode 100644 src/plone/restapi/tests/test_services_groups.py


works for me on the new release.
